bots/image-prune

```diff
@@ -114,7 +114,7 @@ def get_image_names(quiet=False, open_pull_requests=True, offline=False):
         os.chdir(os.path.join(BOTS, ".."))
         refs = get_refs(open_pull_requests, offline)
         # list images present in each branch / pull request
-        for name, ref in refs.items():
+        for name, ref in list(refs.items()):
```

Comment:
This looks like overzealous conversion from 2to3? `for` is perfectly happy with an iterator :-) (harmless of course) There are a few more unnecessary `list()` conversions in this PR which are being used in `for` loops.

Reply:
Fixde.